integration_tests/requirements.txt

```diff
@@ -1 +1 @@
-dbt==0.18.0
\ No newline at end of file
+dbt~=0.20.0
```

Comment:
what's the squiggly mean?

Reply:
It means it will select the latest version of the package, greater than or equal to 0.20.0, but still in the 0.20.* version, so it won't download 0.21.0 for example.